src/Exception/Database/DatabaseExceptionTrait.php

```diff
@@ -7,7 +7,7 @@
     /** @var string */
     protected $driver;
 
-    private static $driverNames = [
+    protected static $driverNames = [
```

Comment:
FWIW I get a chain of the correct 4 exceptions from PDO though to DBAL for a bad DB name.